app-text/ripgrep/ripgrep-0.10.0.recipe

```diff
@@ -0,0 +1,62 @@
+SUMMARY="Combines the usability of ag with the raw speed of grep"
+DESCRIPTION="ripgrep is a line-oriented search tool that recursively searches \
+your current directory for a regex pattern while respecting your gitignore \
+rules. It is similar to other popular search tools like The Silver Searcher, \
+ack and grep."
+HOMEPAGE="https://github.com/BurntSushi/ripgrep"
+COPYRIGHT="2015 Andrew Gallant"
+LICENSE="MIT
+	Public Domain"
```

Comment:
```suggestion
	Unlicense"
```

Our "Public Domain" license is the CC0 (Creative Commons Zero) license, and not the Unlicense. Please add a file named `Unlicense` containing said license text to the `licenses` folder relative to this recipe.